### ISSUE TITLE
Possible typo in pulse_scheduler tutorial?

### ISSUE THREAD

**Issue by Kee-Wang** [Qiskit/qiskit#10676]
This might be a trivial one and I got confused for a sec.

In box 7 of [pulse_scheduler tutorial](https://github.com/Qiskit/qiskit-tutorials/blob/master/tutorials/circuits_advanced/07_pulse_scheduler.ipynb)
'...but in ALAP, qubit 0 has no time to decay from the excited state before measurement.' 
Do you mean 'qubit 1' or am I understanding it incorrectly?

**Comment by abbycross** [comment]
I don't think we support this page - @Eric-Arellano, thoughts on whether to close this or transfer elsewhere?

**Comment by Eric-Arellano** [comment]
Yes, this tutorial is no longer actively maintained. The original post is three years ago. Closing as stale
